README.md

```diff
@@ -2,42 +2,69 @@
 
 [![CircleCI](https://circleci.com/gh/idpass/idpass-lite.svg?style=svg)](https://circleci.com/gh/idpass/idpass-lite)
 
+A library to create and issue biometrically-binding QR code identity cards.
+
 ![Alt text](idpasslite_qr.png?raw=true "api")
 
+## Usage
+
+This library can be used in C and C++ projects. Download `libidpasslite.so` from the [Releases](https://github.com/idpass/idpass-lite/releases) page or [build it from source](#building-from-source), then add it to the project that will use it.
```

Comment:
> Download `libidpasslite.so` from the Releases page

This is actually not yet the case now, but I think it would be nice to provide prebuilt binaries in the releases so that users can just directly download them instead of having to manually build from source. For every release of the library we should attach the corresponding versions of these binaries.

Also do we plan to adapt a versioning system like SemVer at some point?

Reply:
I agree with the `semver`. I will embed semver into binary

Reply:
`semver` is added in https://github.com/idpass/idpass-lite/commit/5033c568460875159b3d12625cfa65f84a3b6476 as `version.txt` and embedded inside binary visible by:

`strings libidpasslite.so | grep TRACKER`